backend/src/config.ts

```diff
@@ -0,0 +1,101 @@
+// The config object that gets passed around to instantiate the server
```

Comment:
No major changes here. Just some stronger typings and some different ways for generating the config.